hack/.deployUtils

```diff
@@ -257,7 +260,10 @@ joinSubmarinerBroker() {
 }
 
 deployThanos() {
-  local THANOS_KUSTOMIZATION_DIR=${LOCAL_SETUP_DIR}/../config/thanos
+  local THANOS_KUSTOMIZATION_DIR="$2"
+  if [ -z "$2" ]; then
```

Comment:
Will these locations ever be different?
If not, it might save you the bother of maintaining the location vars separate in more than 1 file.

Reply:
It would be different for local-setup